usr/src/man/man1m/zfs.1m

```diff
@@ -1630,8 +1630,7 @@ and so forth
 can be used to manipulate both native properties and user properties. Use the
 .Nm zfs Cm inherit
 command to clear a user property . If the property is not defined in any parent
```

Comment:
I noticed that this line's period is off. Dunno if it's worth fixing while here. If not, I can fire up something with that myself.